### ISSUE TITLE
NullPointerException when calling AnnotationInfo#getParameterValues

### ISSUE THREAD

**Issue by torstenkuhnhenne** [classgraph/classgraph#370]
I want to get the current value from the `io.swagger.annotations.ApiOperation` annotation. In my use case I must get the value for `notes`.

```
final AnnotationInfo annotationInfo = methodInfo.getAnnotationInfo( ApiOperation.class.getName() );
assertThat( annotationInfo ).isNotNull();
final String value = annotationInfo.getParameterValues().get( "notes" ).getValue().toString();
```
Annotation to get the value from: `@ApiOperation(value = "", notes = "${snippetclassifications.findById}", response = RestSnippetClassification.class)`

The `getParameterValues()` call throws the following `NullPointerException`:
```
java.lang.NullPointerException
	at io.github.classgraph.ObjectTypedValueWrapper.getArrayValueClassOrName(ObjectTypedValueWrapper.java:291)
	at io.github.classgraph.ObjectTypedValueWrapper.convertWrapperArraysToPrimitiveArrays(ObjectTypedValueWrapper.java:372)
	at io.github.classgraph.AnnotationParameterValue.convertWrapperArraysToPrimitiveArrays(AnnotationParameterValue.java:165)
	at io.github.classgraph.AnnotationParameterValueList.convertWrapperArraysToPrimitiveArrays(AnnotationParameterValueList.java:110)
	at io.github.classgraph.AnnotationInfo.convertWrapperArraysToPrimitiveArrays(AnnotationInfo.java:458)
	at io.github.classgraph.ObjectTypedValueWrapper.convertWrapperArraysToPrimitiveArrays(ObjectTypedValueWrapper.java:362)
	at io.github.classgraph.AnnotationParameterValue.convertWrapperArraysToPrimitiveArrays(AnnotationParameterValue.java:165)
	at io.github.classgraph.AnnotationParameterValueList.convertWrapperArraysToPrimitiveArrays(AnnotationParameterValueList.java:110)
	at io.github.classgraph.AnnotationInfo.getParameterValues(AnnotationInfo.java:136)
	at de.myview.xom.rest.api.AbstractApiControllerTest.checkApiOperationNotesProperty(AbstractApiControllerTest.java:590)
	at de.myview.xom.rest.api.AbstractApiControllerTest.lambda$33(AbstractApiControllerTest.java:578)
```

This happens when the code tries to get the default-value for the property `properties`. This property has following definition:

```
Extension[] extensions() default @Extension(properties = @ExtensionProperty(name = "", value = ""));
```

I have used ClassGraph 4.8.47 and attached the verbose output of my test.
[verbose.log](https://github.com/classgraph/classgraph/files/3656615/verbose.log)

**Comment by lukehutch** [comment]
Hi, can you please create a self-contained unit test that duplicates the behavior, and submit as a PR? I'm currently traveling, but I will look at this ASAP.

**Comment by torstenkuhnhenne** [comment]
Hi @lukehutch, I shared a GitHub repository with a simple test.

**Comment by lukehutch** [comment]
@torstenkuhnhenne I tried pulling in your unit test to the ClassGraph repo (with a couple of changes that should not affect functionality). However the test passes for me. Can you please try getting the test to break in the ClassGraph tree? Thanks.

**Comment by torstenkuhnhenne** [comment]
@lukehutch I have changed the test and now it fails. I have pushed the changes to my fork: https://github.com/torstenkuhnhenne/classgraph

**Comment by lukehutch** [comment]
@torstenkuhnhenne this should be fixed in master. Please test, then I will push out a release.

**Comment by lukehutch** [comment]
PS for the record, to trigger this bug, you had to use an annotation with a default value, and the default value had to be a reference to a class that was not whitelisted (i.e. not encountered during scanning), and/or an array type of references to classes that were not whitelisted.

**Comment by torstenkuhnhenne** [comment]
@lukehutch I can confirm that the current snapshot version is working :-)

Thank you for your quick support and the good work. I use ClassGraph a lot and it works like a charm!

**Comment by lukehutch** [comment]
Great, thanks for confirming! I'll push out a release in a few hours when I get back to the hotel, but unfortunately I won't be able to release a new jar for a while because I'm in process of switching from Sonatype to BinTray for releases, and I'm waiting on a BinTray ticket. (BinTray has been very slow to respond...)

**Comment by lukehutch** [comment]
PS I'm glad you find ClassGraph useful!

**Comment by lukehutch** [comment]
@torstenkuhnhenne After a very long time trying to solve issues with both Sonatype and BinTray, Maven deployment is fixed again, and ClassGraph version 4.8.48 should be in Maven Central in a few hours. Sorry for the delay.
